### PR TITLE
Feat: Added filter functionality in feedback [#1439]

### PR DESCRIPTION
**Pull request by Abhishek9503**
## Description

#1439 
In this PR i have added the functionality of filters in feedback section , filters include search , category and status based filter 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [ ] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

File: src/packages/@support/features/feedback-section/layout/FeedbackSection.svelte [hunk 1]
```diff
@@ -100,9 +119,9 @@
           icon={CategoryIcon}
           onclick={(id = "") => {
             type = id;
+            handleCategoryChange(id);
           }}
           titleId={type}
-          id={"feeds"}
```
Comment (itsmdasifraza) [inline src/packages/@support/features/feedback-section/layout/FeedbackSection.svelte:105]: make this `categoryType`

File: src/packages/@support/features/feedback-section/layout/FeedbackSection.svelte [hunk 2]
```diff
@@ -9,51 +9,70 @@
...
-      );
+  // Function to handle input change
+  const handleInputChange = async (searchQuery: string) => {
+    searchTerm = searchQuery;
+    await getPosts(currentSort, searchQuery, status); // Fetch posts with search term
+  };
+
+  const handleSelectChange = async (_status: string) => {
+    if (_status == "all status") {
+      status = "open,under review,planned,in progress,complete";
```
Comment (itsmdasifraza) [inline src/packages/@support/features/feedback-section/layout/FeedbackSection.svelte:57]: store this in a const string, and use at multiple places

File: src/packages/@support/features/feedback-section/layout/FeedbackSection.svelte (hunk 1 above)
Comment (Abhishek9503) [reply src/packages/@support/features/feedback-section/layout/FeedbackSection.svelte:105]: Resolved

File: src/packages/@support/features/feedback-section/layout/FeedbackSection.svelte (hunk 2 above)
Comment (Abhishek9503) [reply src/packages/@support/features/feedback-section/layout/FeedbackSection.svelte:57]: Resolved

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**